src/Integration/IServiceProviderExtensions.cs

```diff
@@ -58,5 +59,12 @@ public static T GetMefService<T>(this IServiceProvider serviceProvider)
             // We don't want to throw in the case of a missing service (don't use GetService<T>)  
             return componentModel?.GetExtensions<T>().SingleOrDefault();
         }
+
+        [Conditional("DEBUG")]
+        internal static void AssertLocalServiceIsNotNull<T>(this T service)
```

Comment:
This feels a little strange; it applies to all reference types(!).
Might just be clearer to write inline the Debug statement.